src/Sign.Cli/CodeCommand.cs

```diff
@@ -54,6 +61,145 @@ internal CodeCommand()
             AddGlobalOption(VerbosityOption);
         }
 
+        internal async Task Handle(InvocationContext context, IServiceProviderFactory serviceProviderFactory, ISignatureProvider signatureProvider, string fileArgument)
```

Comment:
```suggestion
        internal async Task HandleAsync(InvocationContext context, IServiceProviderFactory serviceProviderFactory, ISignatureProvider signatureProvider, string fileArgument)
```

Reply:
Thanks! I have updated my commit and rebased it.